packages/react-charting/src/components/Legends/Legends.base.tsx

```diff
@@ -421,7 +401,7 @@ export class LegendsBase extends React.Component<ILegendsProps, ILegendState> {
     const { palette } = theme!;
     let legendColor = color;
     // if one or more legends are selected
-    if (this.state.isLegendSelected) {
+    if (this._isLegendSelected) {
```

Comment:
You can use this.state.selectedLegend !== '' || Object.keys(this.state.selectedLegends).length > 0 expression here directly